src/api/orders.rs

```diff
@@ -0,0 +1,119 @@
+use serde::{Deserialize, Serialize};
+
+use crate::{client::Client, error::Error};
+
+#[derive(Serialize, Deserialize, Debug)]
+// The struct for placing an order - body of the post request
+pub struct OrderPlacing {
+    pub isin: String,
+    pub expires_at: Option<String>,
+    pub side: OrderType, // "buy" or "sell"
+    pub quantity: i64,
+    pub venue: Option<String>,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub enum OrderType {
+    buy,
+    sell
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+// The struct for placing an order - the response of the request
+pub struct OrderPlacingResponse<T> {
+    pub time: String,
+    pub status: String,
+    pub mode: String,
+    pub results: Option<T>,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+// The struct for placing an order - the results response of the request
+pub struct OrderResults {
+    pub created_at: String,
```

Comment:
This can be converted to chrono's datetime type.
I.e add chrono prelude at the top, and make this the type:
```rust
DateTime<Utc>
```

Reply:
Ok [`3006db3` fixed here](https://github.com/Uzaaft/Septoria/pull/2/commits/3006db3c9e3277dcde397a5ac31e2cb89732aafd)